test/units/testsuite-15.sh

```diff
@@ -30,17 +32,17 @@ clear_units() {
 }
 
 create_service() {
-    local SERVICE_NAME="${1:?}"
-    clear_units "${SERVICE_NAME}".service
+    local service_name="${1:?}"
+    clear_units "${service_name}".service
 
-    cat >/etc/systemd/system/"$SERVICE_NAME".service <<EOF
+    cat >/etc/systemd/system/"$service_name".service <<EOF
```

Comment:
this is still /etc?

Reply:
Yeah, the commit message was a bit misleading. The original point was to make the test not fail when run multiple times in a row due to left over dropins. We still do want to test if stuff under /etc and /usr works correctly, and the test properly cleans the units/dropins before and after each test; it was just this couple of "global" dropins that weren't cleaned and cause issues. I tweaked and reworded the commit accordingly.